core/tests/protractor_utils/ExplorationEditorImprovementsTab.js

```diff
@@ -62,123 +62,135 @@ var ExplorationEditorImprovementsTab = function() {
   var stateNameLocator =
     by.css('.protractor-test-improvements-task-state-name');
 
-  var newTaskStateNameMatcher = (expectedStateName) => {
-    return (task) => {
+  var newTaskStateNameMatcher = function(expectedStateName) {
```

Comment:
Why are we moving to this convention?

Reply:
This was a convention that I introduced and I decided I really don't like it :( I decided to go back to using a real function b/c it helps distinguish it from the "higher-order function" calls

Reply:
okay